wavefront/server/modules/agents_module/agents_module/services/workflow_events.py

```
@@ -118,19 +74,28 @@ def event_callback(event: AriumEvent) -> None:
-
+            running_loop = loop
+            try:
+                running_loop = asyncio.get_running_loop()
+            except RuntimeError:
+                pass
+
+            if running_loop is not None:
+                asyncio.ensure_future(
+                    event_streamer.add_event(execution_id, event_message),
+                    loop=running_loop,
+                )
```

Ruff (0.15.10)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)